src/avram/database.cr

```diff
@@ -156,7 +156,7 @@ abstract class Avram::Database
   end
 
   # :nodoc:
-  def transaction : Bool
+  def transaction
     if current_transaction
       yield
```

Comment:
Now, as a sort of "hack", we can leave the return type as `Bool`, but then add a `true` right after this line. The `wrap_in_transaction` method will already return the bool for that branch. This lets people put whatever in the block, and we still have that return type. The reason I say "hack" though is, i'm not a huge fan of just have a random static value just chillin there, but It would be a step in the right direction, and no different than what we're doing now 🤔

Reply:
Wouldn't it be better if this method was declared `def transaction : Nil` ? Anything that attempted to use the value would get a complaint.